laze-project.yml

```
@@ -856,6 +856,19 @@ modules:
         FEATURES:
           - ariel-os/coap-server
 
+  - name: coap-server-config-storage
+    help: Configure the CoAP server to accept requests depending on build- and runtime configuration
+    depends:
+      - coap
+      - sw/storage
+    provides_unique: [coap-server-config]
```

```suggestion
    provides_unique:
      - coap-server-config
```

we'll have a convention for that soon